tests/formulas/paje/paje_cmg.yaml

```
@@ -120,3 +120,236 @@
     paje_base: 0.4595 * 411.92
     paje_cmg: 1.7257 * 411.92 + 355.43
     paje: (0.4595 * 411.92) + (1.7257 * 411.92 + 355.43)
+
+
+- name: Après 2023, de 3 à 6 ans, la CMG est pleine
+  period: 2023-02
+  relative_error_margin: 0.01
+  input:
+    individus:
+      parent1:
+        age: 40
+        salaire_imposable: 20000
+        activite: actif
+      parent2:
+        age: 40
+        salaire_imposable: 20000
+        activite: actif
+      enfant1:
+        age: 5
+    familles:
+      famille_0:
+        parents: [parent1, parent2]
+        enfants: [enfant1]
+        en_couple: 1t
```

Typos sur les booléens :
```suggestion
        en_couple: true
```